app.go

```diff
@@ -96,6 +96,7 @@ type Headscale struct {
 	ipAllocationMutex sync.Mutex
 
 	shutdownChan chan struct{}
+	wg           sync.WaitGroup
```

Comment:
Could we give this a more descriptive name? what is the group waiting for?

Reply:
Updated the name. This wait group is for `PollNetMap` streams.